web/views/log_entry.py

```diff
@@ -0,0 +1,129 @@
+from django.shortcuts import get_object_or_404
+from django.views.generic import ListView
+from django.contrib.contenttypes.models import ContentType
+from django.core.exceptions import PermissionDenied
+
+from core.models.user import User
+from core.permissions import constants, CheckerMixin
+from auditlog.models import LogEntry
+from auditlog.registry import auditlog
+import datetime
+import json
+
+class LogEntryListView(CheckerMixin, ListView):
+    DATE_FORMAT = "%Y-%m-%d"
+    model = LogEntry
+    allow_empty = True
+    permission_required = constants.Permissions.PROTECTED
+
+    template_name = 'history/log_entry_list.html'
+
+    # TODO
+    #  - Make cleaner
+    def get_list_of_model_fields(self):
+        fields_per_model = LogEntry.objects.values("content_type", "changes").distinct().all()
+        tmp_dict = {}
+        for entry in fields_per_model:
+            entry_content_type = ContentType.objects.get(pk=entry["content_type"])
+            entry_model = entry_content_type.model_class()
+            entry_model_name = entry_content_type.name
+
+            if entry_model_name not in tmp_dict:
```

Comment:
maybe that's a use case for defaultdict, no?

Reply:
Yes, I modified